R/pkg/R/client.R

```diff
@@ -60,6 +60,15 @@ generateSparkSubmitArgs <- function(args, sparkHome, jars, sparkSubmitOpts, pack
   combinedArgs
 }
 
+determineLauncher <- function(sparkSubmitBin, combinedArgs, capture = FALSE) {
```

Comment:
You can use `nolint` - See `context.R` for an example. BTW what is the lint error here ?